GNUmakefile

```diff
@@ -340,7 +340,7 @@ distclean: clean
 manpages: build-coreutils
 	mkdir -p $(BUILDDIR)/man/
 	$(foreach prog, $(INSTALLEES), \
-		$(BUILDDIR)/coreutils manpage $(prog) > $(BUILDDIR)/man/$(PROG_PREFIX)$(prog).1; \
+		$(BUILDDIR)/coreutils manpage $(prog) | sed 's/^\* *\(.*\)/.IP \\[bu]\n\1/' > $(BUILDDIR)/man/$(PROG_PREFIX)$(prog).1; \
```

Comment:
could you please add a comment explaining the why? :)
thanks

Reply:
Update has been made, I had to change the comment to something less descriptive as the spell check lint was unhappy with clap_mangen

Reply:
@xuky76 an alternative would have been to add the word to the "spell-checker:ignore" line at the top of the file